pyentrp/entropy.py

```diff
@@ -102,60 +101,88 @@ def shannon_entropy(time_series):
     return ent
 
 
-def sample_entropy(time_series, sample_length, tolerance=None):
-    """Calculate and return Sample Entropy of the given time series.
-    Distance between two vectors defined as Euclidean distance and can
-    be changed in future releases
-
+def sample_entropy(time_series, m, tol = None):
+    """Calculates the sample entropy of degree m of a time_series.
+    
+    This method uses chebychev norm and searches iteratively. It is quite fast for random data, but can be slower is there is structure in the time series. 
```

Comment:
Possibly a typo here: `slower if there is a structure`?

Reply:
To my knowledge of english it is OK... A building is a 'a structure'. Patterns are just 'structure'